index.html

```diff
@@ -489,9 +489,19 @@ <h4>Proof Points</h4>
             </ul>
           
             <section id="support-staff-training">
-            <h5>Support Staff Training</h5>
-            <p>Training is in place for support staff to build and maintain relevant skills in support of this dimension&apos;s [=proof points=].</p>
-          </section>
+                <h5>Support Staff Training</h5>
+                <p>Training is in place for support staff to build and maintain relevant skills in support of this dimension&apos;s [=proof points=].</p>
+            </section>
+
+            <section id="support-metrics-goals">
+                <h5>Support Metrics and Goals</h5>
+                <ul>
+                    <li>Establish appropriate / meaningful goals and metrics the support organizations, to measure and track progress towards achieving those goals.</li>
```

Comment:
Looks like there's a typo here—should it be:
```suggestion
                    <li>Establish appropriate / meaningful goals and metrics for the support organizations, to measure and track progress towards achieving those goals.</li>
```

Reply:
Thanks @fstrr all fixed.